dev-lang/vala/vala-0.40.3.recipe

```diff
@@ -0,0 +1,112 @@
+SUMMARY="An object-oriented programming language that uses the GObject system"
+DESCRIPTION="Vala is a new programming language that aims to bring modern \
+programming language features to GNOME developers without imposing any \
+additional runtime requirements and without using a different ABI \
+compared to applications and libraries written in C."
+HOMEPAGE="https://wiki.gnome.org/Projects/Vala"
+COPYRIGHT="2006-2018 Jürg Billeter
+	2006-2018 Raffaele Sandrini"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://download.gnome.org/sources/vala/${portVersion%.*}/vala-$portVersion.tar.xz"
+CHECKSUM_SHA256="6d4f1f0b2edaa0d4aa96f72735a9845e6b1acf80a0a2ae494b5d43e07833119e"
```

Comment:
```diff
-CHECKSUM_SHA256="6d4f1f0b2edaa0d4aa96f72735a9845e6b1acf80a0a2ae494b5d43e07833119e"
+CHECKSUM_SHA256="379354a2a2f7ee5c4d6e0f5e88b0e32620dcd5f51972baf6d90d9f18eb689198"
```
(That's in case you would like to update to vala `0.40.4` which was released yesterday.)